frontend_tests/puppeteer_tests/user-deactivation.ts

```diff
@@ -99,13 +99,26 @@ async function test_bot_deactivation_and_reactivation(page: Page): Promise<void>
     const default_bot_user_row = await user_row(page, "Zulip Default Bot");
 
     await page.click(default_bot_user_row + " .deactivate");
+    await common.wait_for_micromodal_to_open(page);
+
+    assert.strictEqual(
+        await common.get_text_from_selector(page, ".dialog_heading"),
+        "Deactivate Zulip Default Bot",
+        "Deactivate modal has wrong user.",
```

Comment:
I don't think "has wrong user" is necessarily a good choice of message here -- the test code doesn't know why the string doesn't match (Could be the user is the same, but there's now a `.` at the end or whatever).  So I'd instead go with something like "Unexpected title for deactivate bot modal" that clearly communicates what the test has checked.

Reply:
I used this string because we already use it for deactivate user modal.

Reply:
Should I also change the error string for the "Deactivate" and "Reactivate" user modal, similar to what you suggest "Unexpected title for deactivate/reactivate user modal"?